src/solution.c

```diff
@@ -602,8 +599,15 @@ static void solution_thread(void *arg)
         ephemeris_unlock();
       }
 
-      /* Send the observations. */
```

Comment:
It looks like this commit is supposed to 
1. still propagate and post back to the mailbox even if `fabs(t_check - (u32)t_check) < TIME_MATCH_THRESHOLD)` fails
2. additionally check for `TIME_FINE` before sending observations

Am I missing something, or is the commit message inaccurate?